.github/workflows/desktop.yml

```diff
@@ -19,7 +19,11 @@ jobs:
                 node-version: '14.x'
 
             - name: Install node packages
-              run: npm ci -ddd
+              uses: nick-invision/retry@7c68161adf97a48beb850a595b8784ec57a98cbb
+              with:
+                timeout_minutes: 10
+                max_attempts: 10
+                command: npm ci
```

Comment:
did we need `-ddd` ? No idea what that flag does

Reply:
Those are "silly" debug logs, we don't need them anymore with the retry. They were just for investigating